akka-distributed-data/src/main/scala/akka/cluster/ddata/protobuf/ReplicatedDataSerializer.scala

```diff
@@ -327,83 +327,159 @@ class ReplicatedDataSerializer(val system: ExtendedActorSystem)
     }
   }
 
-  def ormapToProto(ormap: ORMap[_]): rd.ORMap = {
+  def ormapToProto(ormap: ORMap[_, _]): rd.ORMap = {
     val b = rd.ORMap.newBuilder().setKeys(orsetToProto(ormap.keys))
-    ormap.entries.toVector.sortBy { case (key, _) ⇒ key }.foreach {
-      case (key, value) ⇒ b.addEntries(rd.ORMap.Entry.newBuilder().
-        setKey(key).setValue(otherMessageToProto(value)))
+    ormap.entries.toVector.foreach {
```

Comment:
It's important to sort the entries in a deterministic order so that the same SHA-1 digests is generated for the same content. I suggest that you do something similar to `gsetToProto`